ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/masq-subnet/provision/cucushift-installer-rehearse-vsphere-ipi-ovn-masq-subnet-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-ovn-masq-subnet-provision
+  steps:
+  - chain: ipi-conf-vsphere
+  - ref: ovn-conf-masq-subnet
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
```

Comment:
Guess you can also create similar chain `cucushift-installer-check` or check the masq-subnet is correct